mmedit/models/editors/disco_diffusion/clip_ext.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Why the file name is 'clip_ext' but the class in it is 'ClipWrapper'.

Reply:
I'll rename the file to `clip_wrapper.py`.